### ISSUE TITLE
Training with less joints

### ISSUE THREAD

**Issue by jacksoncsy** [Nicholasli1995/EvoSkeleton#43]
Hello again,

Sorry for another issue! 

I was trying to modify your code to train with 13 joints (instead of 17) in fully supervised manner, so far it seems to run. There are a few things I would like to consult you: (1) should I use lesser number of stages and blocks? (2) should I train with lower learning rate or fewer epochs? (3) Will the removal of head, thorax and spine affect the performance a lot? (4) I can see that you remove nose in your training, may I know why?

I really appreciate your help! Thanks in advance!

Shiyang

**Comment by Nicholasli1995** [comment]
> Hello again,
> 
> Sorry for another issue!
> 
> I was trying to modify your code to train with 13 joints (instead of 17) in fully supervised manner, so far it seems to run. There are a few things I would like to consult you: (1) should I use lesser number of stages and blocks? (2) should I train with lower learning rate or fewer epochs? (3) Will the removal of head, thorax and spine affect the performance a lot? (4) I can see that you remove nose in your training, may I know why?
> 
> I really appreciate your help! Thanks in advance!
> 
> Shiyang

(1) and (2): It's hard to expect the performance without running. The default network architecture and training setting should work fine. You can see the validation error and choose the best parameters in your case.
(3): Yes. Two few input joints will make the input ambiguous. There can be many 3D poses that can project to the same 2D pose.
(4): I removed it because many in-the-wild 2D pose detectors do not detect this joint.

**Comment by jacksoncsy** [comment]
Thank you for the advice! They are very helpful! Guess that I have to try a couple of settings then.
